src/Microsoft.Extensions.ServiceDiscovery.Yarp/ServiceDiscoveryReverseProxyServiceCollectionExtensions.cs

```diff
@@ -11,7 +10,7 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// <summary>
 /// Extensions for <see cref="IServiceCollection"/> used to register the ReverseProxy's components.
 /// </summary>
-public static class ReverseProxyServiceCollectionExtensions
+public static class ServiceDiscoveryReverseProxyServiceCollectionExtensions
```

Comment:
That's a name...

Reply:
Thankfully, it's just an extension class, so no one needs to actually type this.